common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1083,6 +1083,12 @@
     ],
     "sqlState" : "22008"
   },
+  "DATE_TIME_FIELD_OUT_OF_BOUNDS" : {
+    "message" : [
+      "The value '<badValue>' you entered for <unit> is not valid. Please provide a value between <range>. To disable strict validation, you can turn off ANSI mode by setting <ansiConfig> to false."
```

Comment:
cc @srielau updated the error message including more parameters such as `unit`, `range` and `badValue`